x-pack/test/functional/apps/transform/start_reset_delete/starting.ts

```diff
@@ -5,36 +5,106 @@
  * 2.0.
  */
 
-import { TRANSFORM_STATE } from '@kbn/transform-plugin/common/constants';
+import {
+  TRANSFORM_STATE,
+  TRANSFORM_HEALTH,
+  TRANSFORM_HEALTH_LABEL,
+  TRANSFORM_HEALTH_DESCRIPTION,
+} from '@kbn/transform-plugin/common/constants';
+import {
+  TransformLatestConfig,
+  TransformPivotConfig,
+} from '@kbn/transform-plugin/common/types/transform';
 import { FtrProviderContext } from '../../../ftr_provider_context';
 import { getLatestTransformConfig, getPivotTransformConfig } from '../helpers';
 
+interface TestDataPivot {
+  suiteTitle: string;
+  originalConfig: TransformPivotConfig;
+  mode: 'batch' | 'continuous';
+  type: 'pivot';
+  expected: {
+    healthDescription: string;
+    healthLabel: string;
+    healthStatus: string;
+  };
+}
+
+interface TestDataLatest {
+  suiteTitle: string;
+  originalConfig: TransformLatestConfig;
+  mode: 'batch' | 'continuous';
+  type: 'latest';
+  expected: {
+    healthDescription: string;
+    healthLabel: string;
+    healthStatus: string;
+  };
+}
+
+type TestData = TestDataPivot | TestDataLatest;
+
 export default function ({ getService }: FtrProviderContext) {
   const esArchiver = getService('esArchiver');
   const transform = getService('transform');
 
   describe('starting', function () {
     const PREFIX = 'starting';
-    const testDataList = [
+    const testDataList: TestData[] = [
       {
         suiteTitle: 'batch transform with pivot configuration',
         originalConfig: getPivotTransformConfig(PREFIX, false),
         mode: 'batch',
+        type: 'pivot',
+        expected: {
+          healthDescription: TRANSFORM_HEALTH_DESCRIPTION.green,
+          healthLabel: TRANSFORM_HEALTH_LABEL.green,
+          healthStatus: TRANSFORM_HEALTH.GREEN,
```

Comment:
super duper nit and not a blocker: I know this constant was already defined before this PR but I wonder if we should update 'green', etc. to also be lowercased  or `TRANSFORM_HEALTH_LABEL/TRANSFORM_HEALTH_DESCRIPTION` to use uppercase for consistency?

Reply:
Now fixed in follow up: https://github.com/elastic/kibana/pull/151415